src/actions/auth.js

```diff
@@ -112,21 +112,3 @@ export const refreshAuthToken = () => (dispatch, getState) => {
     });
 };
 //change
-export const logout = (tabClosed = false) => (dispatch, getState) => {
```

Comment:
Prefer to keep the logout action, which is only responsible for dipatching CLEAR_AUTH and telling the local storage to clear the token.